app/main.py

```diff
@@ -1,8 +1,41 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
```

Comment:
When Yurii checked my code he said that it's better to place one/same number of args per line for better readability. So I guess need to fix it.